project_gtd/README.rst

```diff
@@ -0,0 +1,93 @@
+==========
+Todo Lists
+==========
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/14.0/project_gtd
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_gtd
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module implements a simple personal to-do list based on tasks. It adds an editable list of tasks simplified to the minimum required fields in the project application.
+
+The to-do list is based on the *Getting Things Done* methodology. This world-wide used methodology is used for personal time management improvement.
+
+*Getting Things Done* (commonly abbreviated as *GTD*) is an action management method created by David Allen, and described in a book of the same name.
+
+*GTD* rests on the principle that a person needs to move tasks out of the mind by recording them externally. That way, the mind is freed from the job of remembering everything that needs to be done, and can concentrate on actually performing those tasks.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to `Project / Configuration / GTD / Timeboxes` to configure your timeboxes.
+#. Here, your new timebox is empty and not visible yet. You need to add tasks in it to make it visible.
+#. Go to `Project / Tasks` and edit a task to set the 'Timebox' field.
```

Comment:
From ui point of view it has label `Timeframe`:

![image](https://user-images.githubusercontent.com/3512779/124742756-5222f380-df1d-11eb-83b0-cd293b16a66e.png)

I would align the labels. What do you think?

Reply:
ok, I'll align labels. thanks